roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -69,13 +69,15 @@ def run(self, tmp=None, task_vars=None):
                         msg=str(e),
                     )
 
+            if check.changed:
+                r["changed"] = True
```

Comment:
Actually, I remembered why I did it this way. I don't want to clutter up every check result in `-vvv` output with "changed" when it won't matter for most. I think I'll leave it as-is.